File: c/raw/eg/sitting-duck.c
#include <tankode.h>

struct tankode_out sitting_duck(struct tankode_in in)
{
	struct tankode_out out = {0., 0., 0., 0., 0.};
	return out;
}

int main()
{
	struct tankode_id id;
	id.name   = "sitting-duck";
	id.body   = "yellow8";
	id.track  = "orange4";
	id.gun    = "orange5";
	id.bullet = "orange5";
	id.radar  = "grey9";
	id.scan   =	"orange2";
	tankode_run(id, sitting_duck);
	return 0;
}

File: haskell/eg/raw/sitting-duck.hs
import Tankode.Raw

ident :: Id
ident = Id
  { name = "sitting-duck"
  , bodyColour   = "yellow8"
  , trackColour  = "orange4"
  , gunColour    = "orange5"
  , bulletColour = "orange5"
  , radarColour  = "grey9"
  , scanColour   = "orange2"
  }

sittingDuck :: Tankode () -- :: Input -> () -> ((),Output)
sittingDuck input () = ((), Output 0 0 0 0 0)

main :: IO ()
main = run ident sittingDuck ()
